src/Parser/Shaman/Elemental/Modules/Items/EchoesOfTheGreatSundering.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import ItemIcon from 'common/ItemIcon';
+import ItemLink from 'common/ItemLink';
+import { formatPercentage } from 'common/format';
+import ItemDamageDone from 'Main/ItemDamageDone';
+
+class EchoesOfTheGreatSundering extends Analyzer {
+    static dependencies = {
+        combatants: Combatants,
+    };
+
+    BuffedEarthquakeCasts = 0;
+    EchoesProcsCounter = 0;
+    TotalEarthquakeDamage = 0;
+    TotalEarthquakeCasts = 0;
+    
+    estimate_bonus_damage() {
```

Comment:
This can be made a getter - `get estimatedBonusDamage()` and then used with `this.estimatedBonusDamage`.